app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime

```

Comment:
The best practice is to group imports in the following order:
standard library imports
related third party imports
local application/library specific imports
So you can just change the order of imports.